XDR scan abort command (#12060)
demisto/content: Packs/CortexXDR/ReleaseNotes/3_0_1.md

#### Integrations
##### Palo Alto Networks Cortex XDR - Investigation and Response

> - Added the ***xdr-endpoint-scan-abort*** command which cancel the scan of selected endpoints.

Reviewer: ```suggestion
- Added the ***xdr-endpoint-scan-abort*** command which cancels the scan of selected endpoints.
```

> - Fixed an issue where ***xdr-endpoint-scan*** command failed using some of it's arguments.

Reviewer: ```suggestion
- Fixed an issue where the ***xdr-endpoint-scan*** command failed using some of its arguments.
```

> - Fixed an issue where ***xdr-get-endpoints*** command failed using ``alias`` argument.

Reviewer 2: ```suggestion
- Fixed an issue where the ***xdr-get-endpoints*** command failed using the *alias* argument.
```
